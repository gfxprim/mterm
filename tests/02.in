10 10
\n\n\n\n
