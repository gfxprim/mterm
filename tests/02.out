 ----------
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
|          |
 ----------
size 10x10 cursor 4x0
